framework/doc/content/source/postprocessors/FunctionElementAverage.md

```diff
@@ -0,0 +1,13 @@
+# FunctionElementAverage
```

Comment:
funny that the naming convention is different for FUnction vs everything else

For the others it would have been ElementIntegralFunctionPP

Reply:
Maybe I'll be nice and correct this inconsistency :D